src/components/sequencing/Sequences.svelte

```diff
@@ -40,6 +42,8 @@
       selectedSequence = null;
     }
   }
+  $: workspaceActions = $actionDefinitionsByWorkspace[workspaceId ?? -1];
+  $: workspaceActionsCount = Object.keys(workspaceActions ?? {}).length;
```

Comment:
One thing we used to do was define reactive variables above. It feels a little strange to index this at `-1` if `workspaceId` isn't defined. Could we do something like the following and acomplish the same thing?

`workspaceId ? actionDefinitionsByWorkspace[workspaceId] : undefined`?

Reply:
Yep good call, forgot to include reactive variables for those and agreed that your solution is cleaner.

Reply:
Think I often hold off on making these initial `let` definitions for reactive variables as I develop since it is mostly just for type safety but then forget to check through them all after. Svelte 5 will likely change this which will be interesting..